extension.driver.php

```diff
@@ -97,6 +99,19 @@ public function adjustTable($context) {
 		 * Add components for manual entry ordering
 		 */
 		public function addComponents() {
+
+			$pagination = array(
+				'max-rows' => Symphony::Configuration()->get('pagination_maximum_rows', 'symphony'),
+				'current' => (isset($_REQUEST['pg']) && is_numeric($_REQUEST['pg']) ? max(1, intval($_REQUEST['pg'])) : 1)
+				// 'total' => $pgmax
+			);
+
+			Administration::instance()->Page->addElementToHead(
+				new XMLElement('script', 'Symphony.Pagination='.json_encode($pagination), array(
```

Comment:
Thinking about it, it might be simpler to just add this information to the Symphony context using `Symphony.Context.set()`.